.circleci/config.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
Why make it multi-doc?

Reply:
This header does not make it multi-doc, it is just a document start marker that does not do any harm and pleases my IDE because I have a lint tool that requires document start markers. https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.document_start